docs/textinput.md

```diff
@@ -105,6 +105,29 @@ Inherits [View Props](view.md#props).
 
 ---
 
+### `accessibilityErrorMessage`
+
+String read by screenreaders to indicate an error state of a TextInput.
+Setting accessibilityInvalid to true activates the error message.
```

Comment:
Can we also reference the web specification about this? Or that this follows the web spec? Or does that make things more unclear since it's not exactly following `aria-errormessage`?